prometheus.yml

```
@@ -0,0 +1,166 @@
+    metrics_path: /metrics
+    scrape_interval: 15s
+    scrape_timeout: 10s
+
+  # Blackbox Exporter - مراقب الصندوق الأسود
+  - job_name: 'blackbox'
+    metrics_path: /probe
+    params:
+      module: [http_2xx]
+    static_configs:
+      - targets:
+        - https://google.com
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)